core/ssh_keys.go

```diff
@@ -0,0 +1,103 @@
+package core
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/krystal/go-katapult"
+)
+
+type AuthSSHKey struct {
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitempty"`
+	Fingerprint string `json:"fingerprint,omitempty"`
+}
+
+type SSHKeysClient struct {
+	client   RequestMaker
+	basePath *url.URL
+}
+
+func NewSSHKeysClient(rm RequestMaker) *SSHKeysClient {
+	return &SSHKeysClient{
+		client:   rm,
+		basePath: &url.URL{Path: "/core/v1/"},
+	}
+}
+
+type sshKeysResponseBody struct {
```

Comment:
This will need a `Pagination` field with a json tag for `pagination`, or we won't be able to pickup pagination data from the response.